cmd/nirvana/project/init.go

```diff
@@ -226,13 +234,31 @@ func (o *initOptions) templates(proj string) map[string]string {
 		"pkg/apis/api.go":                        o.templateAPI(),
 		"pkg/message/message.go":                 o.templateMessage(),
 		"pkg/version/version.go":                 o.templateVersion(),
-		"Gopkg.toml":                             o.templateGopkg(),
+		pkgConfPath:                              templateGopkg(),
 		"Makefile":                               o.templateMakefile(),
 		project.DefaultProjectFileName:           o.templateProject(),
 		"README.md":                              o.templateReadme(),
 	}
 }
 
+func (o *initOptions) templatePackageManager() (string, func() string) {
+	switch o.PackageManager {
+	case "mod":
+		return "go.mod", func() string {
```

Comment:
Why don't you just `return "go.mod", o. templateGomod `?

Reply:
2333333333